documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -728,6 +728,28 @@ minIdle = 1
 ----
 ====
 
+You can pass even an empty `hotrodclient.properties`.
+In this case the default HotRod properties of Infinispan will be applyied.
+See http://infinispan.org/docs/stable/user_guide/user_guide.html#hotrod:java-client[Infinispan - Java Hot Rod client].
+
+The only exceptions are these properties:
+
+|===
+| HotRod client property | Hibernate OGM mandatory value
+| infinispan.client.hotrod.marshaller
+| org.hibernate.ogm.datastore.infinispanremote.impl.protostream.OgmProtoStreamMarshaller
+| infinispan.client.hotrod.force_return_values
+| true
+|===
+
+That Hibernate OGM will provide for you if you don't declare them, overriding the Infinispan default.
+
+[IMPORTANT]
+====
+These values for these properties are mandatory in order to keep the expected behaviour of the dialect.
+Then if you try to change these, an exception will be thrown.
+====
+
```

Comment:
It's not bad but can we use a different structure?
Something like:

The  `hotrodclient.properties` is optional, in this case Hibernate OGM will use the following:
...
Every other property will use the default values defined by Infinispan [link]

Let me know what you think about it. If you don't agree, feel free to merge it as it is (with the syntax errors fixed, please).

Reply:
I will apply the change you suggested.